From e970c32e9be8777e72c828cf912067810eab916b Mon Sep 17 00:00:00 2001
From: Raju Kumar Pothuraju <rajukumar.pothuraju@amd.com>
Date: Fri, 21 Jul 2023 14:59:42 +0530
Subject: [DTG_PATCHES 1/3] mipi_csi2_rx: Use isppipeline_in as remote-endpoint

isppipeline tcl updated to have isppipeline_in and _out so updating the
same in mipi_csi2_rx tcl.

Signed-off-by: Raju Kumar Pothuraju <rajukumar.pothuraju@amd.com>
---
 mipi_csi2_rx/data/mipi_csi2_rx.tcl | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/mipi_csi2_rx/data/mipi_csi2_rx.tcl b/mipi_csi2_rx/data/mipi_csi2_rx.tcl
index 787dc76..98d25dc 100644
--- a/mipi_csi2_rx/data/mipi_csi2_rx.tcl
+++ b/mipi_csi2_rx/data/mipi_csi2_rx.tcl
@@ -138,6 +138,11 @@ proc generate {drv_handle} {
 							hsi::utils::add_new_dts_param "$mipi_node" "remote-endpoint" $connectip$drv_handle reference
 							gen_axis_switch_in_remo_endpoint $drv_handle "$connectip$drv_handle"
 						}
+
+					} elseif {[string match -nocase [get_property IP_NAME $connectip] "ISPPipeline_accel"]} {
+						set isppipeline_node [add_or_get_dt_node -n "endpoint" -l isppipeline_in$connectip -p $port_node]
+						hsi::utils::add_new_dts_param "$isppipeline_node" "remote-endpoint" $connectip$drv_handle reference
+
 					} else {
 					set csi_rx_node [add_or_get_dt_node -n "endpoint" -l mipi_csirx_out$drv_handle -p $port_node]
 					gen_endpoint $drv_handle "mipi_csirx_out$drv_handle"
-- 
2.17.1

